cli/src/chipmunk_runner.rs

```diff
@@ -1,10 +1,13 @@
+//! Module to manage running Chipmunk after being built.
+
 use std::{io, process::ExitStatus};
 
 use tokio::process::Command;
 
 use crate::{dev_tools::DevTool, target::Target};
 
-pub async fn run_app() -> io::Result<ExitStatus> {
+/// Runs Chipmunk application from its electron path.
+pub async fn run_chipmunk() -> io::Result<ExitStatus> {
```

Comment:
I would prefer 'run_app', this cli tool may someday be used for other projects as well.